lib/Mojo/Pg/Migrations.pm

```diff
@@ -186,6 +197,19 @@ the caller class and L</"name">.
   -- 1 down
   DROP TABLE messages;
 
+=head2 from_dir
+
+  $migrations = $migrations->from_dir('/home/sri/migrations');
+
+Extract migrations from a directory tree where each versioned migration is in a directory, named for the version, and
+each migration has one or more of C<up.sql>, or C<down.sql> files.
```

Comment:
"one or more of"? May I recommend "one or both of"?

(unless we're looking toward having scripts that can be run between migration phases too, which would add lots of complexity probably)

Reply:
Agreed, that sounds weird.

Reply:
reworded